### PR TITLE
feat: web styles migrations schematics

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/web-styles-migrations/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+```bash
+schematics --help
+```
+
+### Unit Testing
+
+`npm run test` will run the unit tests, using Jasmine as a runner and test framework.
+
+### Publishing
```
Comment (gfellerph) [inline packages/web-styles-migrations/README.md:19]: This section needs to be updated for the changesets publishing workflow.

File: packages/web-styles/src/mixins/_forms.scss [hunk 2]
```diff
@@ -85,31 +85,6 @@
       }
     }
   }
-
-  .custom-control-input {
```
Comment (gfellerph) [inline packages/web-styles/src/mixins/_forms.scss:89]: Why was this removed?

Comment (alizedebray) [reply packages/web-styles/src/mixins/_forms.scss:89]: I removed this because it was dead code: CWF no longer offers `.custom-control-input`'s in v5

File: packages/web-styles-migrations/README.md (hunk 1 above)
Comment (alizedebray) [reply packages/web-styles-migrations/README.md]: I just updated the README

**Review by gfellerph — APPROVED** [review]
Thank you very much, great work!

File: .changeset/violet-snakes-design.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+'@swispost/web-styles-migrations': major
+'@swisspost/web-styles': patch
+---
+
+Created a new package offering Migration Schematics for the Web Styles
+Removed custom input styles that are no longer used
```
Comment (alizedebray) [inline .changeset/violet-snakes-design.md:7]: @gfellerph 
Just added the changeset.
